data/fixtures/scopes/java/argument.actual.iteration.scope

```diff
@@ -0,0 +1,10 @@
+myFunk(5, "hello");
+---
+
+[Range] = 0:7-0:17
+         >----------<
+0| myFunk(5, "hello");
+
+[Domain] = 0:6-0:18
+        >------------<
+0| myFunk(5, "hello");
```

Comment:
I don't believe this is what we usually do, is it? I think we pretty much universally exclude the delimiter pair from iteration scope, but maybe I'm wrong?

Reply:
https://github.com/cursorless-dev/cursorless/blob/189813994fbe5a4a482f49b481427a8937ec781c/data/fixtures/scopes/javascript.core/argument.actual.iteration.scope

Reply:
oh wow super weird. shall we just leave it in both places then? I could go either way, but agree consistency is most important

Reply:
I think we should leave it for now and then later we might actually evaluate the benefits of doing this and make all languages consistent? It's not like either of us have had a problem with this on javascript?